engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Polyglot.java

```diff
@@ -31,6 +31,12 @@ private void createPolyglot(Language language, ModuleScope scope) {
     scope.registerMethod(polyglot, "get_array_size", GetArraySizeMethodGen.makeFunction(language));
     scope.registerMethod(
         polyglot, "is_language_installed", IsLanguageInstalledMethodGen.makeFunction(language));
+    scope.registerMethod(
+        polyglot, "get_executable_name", GetExecutableNameMethodGen.makeFunction(language));
+    scope.registerMethod(
+        polyglot, "get_source_location", GetSourceLocationMethodGen.makeFunction(language));
+    scope.registerMethod(
+        polyglot, "has_source_location", HasSourceLocationMethodGen.makeFunction(language));
```

Comment:
The corresponding shadow definitions in `Builtins.enso` for these three and for `Runtime.primitive_get_stack_trace` seem to be missing.

It is also not completely clear to me what does it mean to get a source location from a polyglot object. Does every object have it? Or does this only apply to stack trace elements? From the usage it seems like it is the latter, but it's unclear as it's not described. Also, if it is the latter, what is the reason that these methods must be exposed as methods of the `Polyglot` module, can't we just use some getters of the stack trace elements (I assume we probably can't but would love to know why)?

Reply:
> Does every object have it? Or does this only apply to stack trace elements?
Ok, by reading Truffle docs I answered this question.

Still, the docs for the builtins would be useful